img/bridge-services/matrix.svg

```diff
@@ -0,0 +1,25 @@
+<svg enable-background="new 0 0 512.005 512.005" version="1.1" viewBox="0 0 512 512" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><g clip-rule="evenodd" fill-rule="evenodd">
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Something is wrong.

please use the `build/image-optimization.sh` of server

Reply:
That's what I used. I manually removed the blanks.

Reply:
Weird, okay then